test/language/arguments-object/10.6-11-b-1.js

```diff
@@ -26,26 +27,11 @@ description: >
 
             var argObj = (function () { return arguments })(1);
 
-            var verifyValue = false;
-            verifyValue = (argObj[0] === 1);
+verifyProperty(argObj, "0", {
```

Comment:
What's up with the indentation in these files? Feel free to also de-indent the code before this. Same elsewhere in this commit.

Reply:
I didn't bother to reformat all files when I replaced `runTestCase` in #415 :smile:

Reply:
I've added 6070e8127bb79e8e5096436422abee8c12324d04 to remove excess whitespace in a couple of more files.